test/errors/invalidNew-detailed.good

```diff
@@ -0,0 +1,16 @@
+─── error in invalidNew.chpl:3 [MemManagementNonClass] ───
+  Cannot use memory management strategy 'owned' with non-class type 'E'.
+      |
+    3 | var y = new owned E();
+      |                   ⎺    
+      |
+  Memory management strategies can only be used with classes.
+
+─── error in invalidNew.chpl:3 [InvalidNew] ───
+  Invalid use of 'new' with type 'E', which is neither a class nor a record.
+      |
+    3 | var y = new owned E();
+      |                   ⎺    
+      |
+  The 'new' expression can only be used with records or classes.
```

Comment:
I wonder whether, for this case, the second error could squash the first one?  Which is to say, if I'm working through the errors one by one, the use of `owned` vs. no ownership vs. whatever is irrelevant given that I really shouldn't have typed the `new` in the first place.  I'm imagining wrestling with the first error, somehow silencing it, and then moving onto the second one and thinking "D'oh!"   If both errors needed to be printed for some reason, I'd much rather see the second first, since that's the root of the problem and the first thing I should fix.  (But then if I saw the other one next, I'd think the compiler wasn't being as smart as it could be).